src/grandorgue/dialogs/GOOrganDialog.cpp

```diff
@@ -336,49 +343,29 @@ GOOrganDialog::GOOrganDialog(
   m_ReleaseLoad->SetSelection(wxNOT_FOUND);
   m_LastReleaseLoad = m_ReleaseLoad->GetSelection();
   box1->Add(grid, 0, wxEXPAND | wxALL, 5);
-  settingSizer->Add(box1, 0, wxEXPAND | wxALL, 5);
-
-  wxBoxSizer *buttons = new wxBoxSizer(wxHORIZONTAL);
-  m_Apply = new wxButton(scroll, ID_EVENT_APPLY, _("Apply"));
-  m_Reset = new wxButton(scroll, ID_EVENT_RESET, _("Reset"));
-  m_Default = new wxButton(scroll, ID_EVENT_DEFAULT, _("Default"));
-  m_DefaultAll
-    = new wxButton(scroll, ID_EVENT_DEFAULT_ALL, _("Default for All"));
-  buttons->Add(m_Default);
-  buttons->Add(m_DefaultAll);
-  buttons->Add(m_Reset);
-  buttons->Add(m_Apply);
-  settingSizer->Add(buttons);
-
-  box1 = new wxStaticBoxSizer(wxVERTICAL, scroll, _("Sample information"));
-  grid = new wxFlexGridSizer(2, 5, 5);
-  box1->Add(grid, 0, wxEXPAND | wxALL, 5);
-
-  grid->Add(
-    new wxStaticText(scroll, wxID_ANY, _("Memory usage:")),
-    0,
-    wxALIGN_RIGHT | wxALIGN_CENTER_VERTICAL | wxBOTTOM,
-    5);
-  m_MemoryDisplay = new wxStaticText(scroll, wxID_ANY, wxEmptyString);
-  grid->Add(m_MemoryDisplay);
-
-  grid->Add(
-    new wxStaticText(scroll, wxID_ANY, _("Bits per sample:")),
-    0,
-    wxALIGN_RIGHT | wxALIGN_CENTER_VERTICAL | wxBOTTOM,
-    5);
-  m_BitDisplay = new wxStaticText(scroll, wxID_ANY, wxEmptyString);
-  grid->Add(m_BitDisplay);
+  mainSizer->Add(
+    box1, wxGBPosition(2, 1), wxGBSpan(1, 4), wxEXPAND | wxRIGHT, 5);
 
-  settingSizer->Add(box1, 0, wxEXPAND | wxALL, 5);
+  m_AudioGroupAssistant = new wxButton(
+    this, ID_EVENT_AUDIO_GROUP_ASSISTANT, _("Distribute audio groups"));
+  mainSizer->Add(
+    m_AudioGroupAssistant, wxGBPosition(4, 0), wxDefaultSpan, wxLEFT, 5);
+  m_Default = new wxButton(this, ID_EVENT_DEFAULT, _("Default"));
+  mainSizer->Add(m_Default, wxGBPosition(4, 1));
+  m_DefaultAll = new wxButton(this, ID_EVENT_DEFAULT_ALL, _("Default for All"));
+  mainSizer->Add(m_DefaultAll, wxGBPosition(4, 2));
+  m_Reset = new wxButton(this, ID_EVENT_RESET, _("Reset"));
+  mainSizer->Add(m_Reset, wxGBPosition(4, 3));
+  m_Apply = new wxButton(this, ID_EVENT_APPLY, _("Apply"));
+  mainSizer->Add(m_Apply, wxGBPosition(4, 4), wxDefaultSpan, wxRIGHT, 5);
 
-  scroll->SetSizer(settingSizer);
-  scroll->SetScrollbars(0, 5, 0, 15);
-  mainSizer->Add(scroll, 1, wxEXPAND);
+  LayoutWithInnerSizer(mainSizer);
+}
 
+bool GOOrganDialog::TransferDataToWindow() {
   FillTree();
-  Load();
-  LayoutWithInnerSizer(mainSizer);
+  // Load();
```

Comment:
Is this commended code intended to be used?

Reply:
No. FillTree calls m_Tree->SelectItem() that calls Load(). 

I removed the line.